pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -261,9 +261,7 @@ type FilterPlugin interface {
 	Plugin
 	// Filter is called by the scheduling framework.
 	// All FilterPlugins should return "Success" to declare that
-	// the given node fits the pod. If Filter doesn't return "Success",
-	// please refer scheduler/algorithm/predicates/error.go
-	// to set error message.
+	// the given node fits the pod. 
```

Comment:
sgtm, looks like those errors has been defined in each of the plugin instead.

btw, it's would be cool to update those line as well?
https://github.com/kubernetes/kubernetes/blob/43fbe17dc682b703b0b663935a797244378f0aaf/pkg/scheduler/framework/v1alpha1/interface.go#L256

`UnschedulableAndUnresolvable` is one of the code that would return from filter plugin.